yt/testing.py

```diff
@@ -360,7 +360,7 @@ def fake_tetrahedral_ds():
     return ds
 
 
-def fake_hexahedral_ds():
+def fake_hexahedral_ds(fields=()):
```

Comment:
Empty tuples as default values are bad practice
```suggestion
def fake_hexahedral_ds(fields=None):
```